docs/product-getting-started-guide/src/main/asciidoc/data-object-driver-create-proc.adoc

```diff
@@ -0,0 +1,84 @@
+[id='data-object-driver-proc']
+= Creating the Driver data object
+
+The following chapter instructs you how to create the Driver data object containing data fields, such as: Name, Age, and Total Points.
+
+//a traffic violation data model in {PRODUCT}. Data models are models of data objects. A data object is a custom complex data type (for example, a Violation data object containing data fields Violation Date, Fine Amount, and Speed Limit).
+
+[float]
+== Prerequisites
+
+You have created the Driver department traffic violations project.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Data Object*.
+. In the *Create new Data Object* wizard, enter the following values:
+.. *Data Object*: `Driver`.
+.. *Package*: select `org.jboss.example.traffic_violations`.
+. Click *+Ok*.
++
+
+.Create new Data Object window
+image::data-object2.png[]
+
+== Adding the Driver data object data fields
+
```

Comment:
See my comment in the Adding the Violation data object data fields section, about intro statements and procedure heading.

Reply:
done